frontend/js/websockets/positionConnector.js

```diff
@@ -10,6 +10,9 @@ socket.onmessage = function (event) {
   if (msg.includes("MOVE")) {
     console.log(msg);
     pos = msg.split(/\s+/);
+    if (pos[4] != "") {
+      updateMap(pos[5], pos[6], pos[4]);
+    }
```

Comment:
Maybe, there should be, like, "CLOSE" command? Or "UPD" command?
Moreover, I don't think, that "MOVE" command should close doors. "MOVE" command should move our player, nothing more, nothing less.